### PR TITLE
Clean up functional selection

### PR DESCRIPTION
**Pull request by vanderhe**
Follow-up to the feedback of #75.

### CODE REVIEW

**Review by bhourahine — APPROVED** [review]
Generally fine, but one question.

File: slateratom/lib/dft.f90 [hunk 1]
```diff
@@ -210,7 +210,7 @@ subroutine density_grid(pp, max_l, num_alpha, poly_order, alpha, num_mesh_points
     case(xcFunctional%HYB_PBE0)
       call getExcVxc_HYB_PBE0(abcissa, dz, dzdr, rho, drho, sigma, camAlpha, exc, vxc)
     case(xcFunctional%HYB_B3LYP)
-      call getExcVxc_HYB_B3LYP(abcissa, dz, dzdr, rho, drho, sigma, exc, vxc)
+      call getExcVxc_HYB_B3LYP(abcissa, dz, dzdr, rho, drho, sigma, camAlpha, exc, vxc)
```
[the remark this answers is not in the document]
Comment (vanderhe) [reply slateratom/lib/dft.f90:213]: I wouldn't say so, it is merely a preparation in anticipation of a generalization of the B3LYP hybrid to arbitrary $\alpha$. Previously $\alpha = 0.2$ was hardcoded in the `getExcVxc_HYB_B3LYP` routine, now it is hardcoded (same value) in the `read_input_1` routine, leaving the behavior/input unchanged. However, the generalization to arbitrary $\alpha$ is facilitated and would only require the `sktools` to hand it over for B3LYP as well and `slateratom` to actually read in its value.

[the remark this answers is not in the document]
Comment (bhourahine) [reply slateratom/lib/dft.f90:213]: Might be a good idea to put something about the change in the log message, so just a git commit --amend and something about preparation for alpha being user supplied.

[the remark this answers is not in the document]
Comment (vanderhe) [reply slateratom/lib/dft.f90:213]: Good idea, done.

**Review by bhourahine — APPROVED** [review]
LGTM

**Pull request merged**